newIDE/app/src/Profile/AuthenticatedUserProfileDetails.js

```diff
@@ -61,6 +61,11 @@ const AuthenticatedUserProfileDetails = ({
         isAuthenticatedUserProfile
         onChangeEmail={onChangeEmail}
         onEditProfile={onEditProfile}
+        disableChangeEmailButton={
```

Comment:
we're already passing down the profile to the `ProfileDetails` component, so you could probably just use it to do the logic inside the component?

Reply:
Good idea, I'll move it